src/bidiMapper/scripts/eval.spec.ts

```diff
@@ -0,0 +1,162 @@
+import fs from 'fs/promises';
+import path from 'path';
+import * as chai from 'chai';
+import chaiExclude from 'chai-exclude';
+
+chai.use(chaiExclude);
+
+describe('Evaluator', function () {
+    let EVALUATOR: any;
+
+    // Get EVALUATOR.
+    before(async function () {
+        const eval_text = await fs.readFile(
+            path.join(__dirname, './eval.es'),
+            'utf8'
+        );
+
+        EVALUATOR = eval(eval_text);
+    });
+
+    function checkSerializeAndDeserialize(originalObject: any, expectedSerializedObj: any, excluding: string[] = []) {
```

Comment:
Minor: Once #44 is merged and `RemoteValue` is fully defined, we can use it for `expectedSerializedObj`.

Reply:
Done in #50